learningequality/studio
# Remove vuetify-loader.

## Summary
### Description of the change(s) you made
* Removes vuetify loader
* Replaces it with explicit imports
* This seems to prevent memory leaks or at least greatly reduce them
* It also seems to make the webpack build an order of magnitude faster
* Updates pre-commit config and frontend linting commands to prevent errors being silently swallowed during precommit.

### Manual verification steps performed
1. Run the devserver
2. Navigate around the site a bit
3. Confirm that there are no warnings of unregistered components or directives
